interfaces.go

```diff
@@ -94,6 +96,26 @@ type MultiTimeSeriesTable interface {
 	TableChanger
 }
 
+type FlakeSeriesTable interface {
+	Set(v interface{}) (Op, error)
+	Update(id string, m map[string]interface{}) (Op, error)
+	Delete(id string) (Op, error)
+	Read(id string, pointer interface{}) (Op, error)
```

Comment:
Gocassa's table accessors to not return errors. Can these errors be deferred until `Run()` is called on the `Op`?